be/src/common/config.h

```diff
@@ -305,11 +305,21 @@ CONF_mInt32(base_compaction_trace_threshold, "120");
 CONF_mInt32(cumulative_compaction_trace_threshold, "60");
 CONF_mInt32(update_compaction_trace_threshold, "20");
 
+// Max columns of each compaction group.
+// If the number of schema columns is greater than this,
+// the columns will be divided into groups for vertical compaction.
+CONF_Int64(vertical_compaction_max_columns_per_group, "5");
```

Comment:
could the number can be judged automatically without add configure?

Reply:
I will improve it later